gobblin-api/src/main/java/org/apache/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -207,6 +207,8 @@
   public static final String CLEANUP_OLD_JOBS_DATA = "cleanup.old.job.data";
   public static final boolean DEFAULT_CLEANUP_OLD_JOBS_DATA = false;
   public static final String MAXIMUM_JAR_COPY_RETRY_TIMES_KEY = JOB_JAR_FILES_KEY + ".uploading.retry.maximum";
+  public static final String USER_DEFINED_STAGING_DIR = "user.defined.staging.dir";
```

Comment:
I'll just name it as "user.defined.static.staging.dir". Let's sync offline for why I suggest doing this.

Reply:
I have renamed key: USER_DEFINED_STAGING_DIR as USER_DEFINED_STATIC_STAGING_DIR and value: user.defined.staging.dir as user.defined.static.staging.dir